### PR TITLE
[139] added tooltip on hover

### PR DESCRIPTION
**Pull request by faizanu94**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/symbols/Label/Label.tsx [hunk 1]
```diff
@@ -21,10 +21,12 @@ export const Label: FC<Partial<LabelProps>> = ({
   className,
   originalText
 }) => (
-  <g transform={`translate(${x}, ${y})`}>
+  <>
```
[the remark this answers is not in the document]
Comment (faizanu94) [reply src/symbols/Label/Label.tsx:24]: it will be for the same node

[the remark this answers is not in the document]
Comment (amcdnl) [reply src/symbols/Label/Label.tsx:24]: What about nested nodes? Did you try that? I think we might not want this in that case.

[the remark this answers is not in the document]
Comment (faizanu94) [reply src/symbols/Label/Label.tsx:24]: the basic example is for a nested one - anyways if tooltip is not required I will work on some other feature and let's ignore this pull request :)

[the remark this answers is not in the document]
Comment (amcdnl) [reply src/symbols/Label/Label.tsx:24]: Alright - I'm totally happy to accept this just want to make sure it works as expected for the edge cases.

[the remark this answers is not in the document]
Comment (faizanu94) [reply src/symbols/Label/Label.tsx:24]: yeah sure, awaited for your review and go ahead :)

[the remark this answers is not in the document]
Comment (faizanu94) [reply src/symbols/Label/Label.tsx:24]: let me know if any further handling required for this pull request

[the remark this answers is not in the document]
Comment (amcdnl) [reply src/symbols/Label/Label.tsx:24]: Can you confirm if when you hover a nested node's children, it doesn't show the parent tooltip?

[the remark this answers is not in the document]
Comment (amcdnl) [reply src/symbols/Label/Label.tsx:24]: Perfect - I'll merge :D

**Comment by amcdnl** [comment]
Any other changes @faizanu94 you want ot make before i release this?

**Comment by faizanu94** [comment]
None as of now - I'll pick and start something soon :) Thanks

File: src/symbols/Label/Label.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (gajjardarshithasmukhbhai) [reply src/symbols/Label/Label.tsx:24]: @faizanu94 @amcdnl @VHalushko How I will modify my tooltip styling?

**Comment by kevinreber** [comment]
Is there a way to tweak this logic so we can add a custom tooltip?
I created a custom node using the `<foreignObject />` approach and tried adding a custom tooltip, but I don't think atm `reaflow` allows custom tooltips

for example
```tsx
import { NodeProps, Node } from "reaflow";

const CustomTooltip = () => {
 // Custom Tooltip component
}

export default const CustomNode = (nodeProps: NodeProps) => {

   return (
    <Node
         {...nodeProps}
         onClick={handleClick as any}
       >
         {({ node, x, y }) => {
           return (
             <CustomTooltip>
               <foreignObject
                 x={0}
                 y={0}
                 width={node.width}
                 height={node.height}
               >
                 <span>{node.id}</ span>
               </ foreignObject>
             </ CustomTooltip>
           );
         }}
    </Node>
   )
}
```

**Pull request merged**